src/main/java/seedu/address/model/budget/BudgetList.java

```diff
@@ -44,6 +51,15 @@ public void remove(Budget toRemove) {
         if (!internalList.remove(toRemove)) {
             throw new BudgetNotFoundException();
         }
+        budgets.remove(toRemove);
+    }
+
+    public List<Expenditure> getExpenditure(int index) {
+        return budgets.get(index).getExpenditures();
```

Comment:
Perhaps it will be more OOP if there is an additional private method to get the the budget of a certain index.